inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/mysql-cdc/src/main/java/org/apache/inlong/sort/cdc/mysql/source/assigners/MySqlSnapshotSplitAssigner.java

```diff
@@ -228,7 +228,10 @@ private void startAsynchronouslySplit() {
                                                 .map(MySqlSnapshotSplit::toSchemaLessSnapshotSplit)
                                                 .collect(Collectors.toList());
                                 synchronized (lock) {
-                                    remainingSplits.addAll(schemaLessSnapshotSplits);
+                                    int size = schemaLessSnapshotSplits.size();
```

Comment:
I guess the steps can be held inside one method. ?

Reply:
done.